setup_dev.sh

```diff
@@ -31,7 +31,9 @@ sed -i "s@^  :password.*@  :password => '$mysql_congress_forms',@" config/databa
 
 HOME=/home/vagrant sudo -u vagrant /bin/bash <<EOF
 echo "Setting up RVM and Ruby..."
-curl -sSL https://get.rvm.io | bash -s stable
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
```

Comment:
The long ID doesn't provide good security, we should use the whole fingerprint if possible.